File: tests/fixtures/multi/example.test.ts
// blindfold multi-language fixture (TypeScript)
function slug(s: string): string { return s.toLowerCase().replace(/ /g, "-"); }

it("justified", () => {
  expect(slug("A B")).toBe("a-b"); // blindfold: doc — slug lowercases, spaces -> dash
});

it("unjustified", () => {
  expect(slug("Hi There")).toBe("hi-there");
});

it("confessed", () => {
  expect(slug("X Y")).toBe("x-y"); // blindfold: snapshot — recorded from a run
});

File: tests/fixtures/multi/example_test.go
// blindfold multi-language fixture (Go)
package example

import "testing"

func add(a, b int) int { return a + b }

func TestJustified(t *testing.T) {
	if got := add(2, 3); got != 5 { // blindfold: math — 2+3 is 5
		t.Fatalf("got %d", got)
	}
}

func TestUnjustified(t *testing.T) {
	if got := add(10, 20); got != 30 {
		t.Fatalf("got %d", got)
	}
}
